infrastructure/main.tf

```diff
@@ -0,0 +1,11 @@
+module "backend" {
+  source   = "git@github.com:contino/moj-module-webapp?ref=master"
+  product  = "${var.product}-backend"
+  location = "${var.location}"
+  env      = "${var.env}"
+  ilbIp    = "${var.ilbIp}"
+
+  app_settings = {
+    S2S_URL = "${var.s2s_url}"
```

Comment:
should this not be something like
```
"http://idam-s2s-${var.env}.service.${data.terraform_remote_state.core_apps_compute.ase_name[0]}.internal"
```
?

Reply:
I don't know. Where can I find any details about this?